mule-packager/src/test/java/org/mule/tools/api/verifier/ProjectVerifierFactoryTest.java

```diff
@@ -43,4 +51,13 @@ public void createPreparePackagerTest() {
       }
     }
   }
+
+  @Test
+  @Issue("W-12354025")
+  @Description("Verify that PolicyYamlVerifier validation works after upgrading to snakeyaml 2.0")
+  public void verifyValidation() throws ValidationException {
+    PolicyYamlVerifier policyYamlVerifier =
+        new PolicyYamlVerifier(this.getClass().getResource("/org/mule/tools/api/verifier").getPath(), "custom.policy.test.yaml");
```

Comment:
What is the logic of this test?
Add a comment explaining it.

Reply:
I had to change PolicyYamlVerifier.validate() as in snakeyaml 2.0 the constructor of Yaml class changed. This is a test to validate we can correctly validate the yaml after this change and it is still able to read fields from the Yaml